### ISSUE TITLE
I installed silent patch and bully still crashing, HELP

### ISSUE THREAD

**Issue by ghost** [CookiePLMonster/SilentPatchBully#62]
I installed the patch and i think it's correctly installed, but the game crash with about 3-10 minutes of gameplay

Here is the dump file, everytime the game crashes i get these dump files

[Bully.exe.20201120153927.zip](https://github.com/CookiePLMonster/SilentPatchBully/files/5575486/Bully.exe.20201120153927.zip)

**Comment by isthejhon** [comment]
I can't even do only one mission because the game crashes, help please

**Comment by ghost** [comment]
the game just crashes when i'm doing missions, only missions

**Comment by ghost** [comment]
man common please help me, the game literally crashed with 10 seconds of gameplay

**Comment by RinMaru** [comment]
I think this project is dead and i think this is more a local issue so best of luck. maybe try limiting the game to 4 cores or 2

**Comment by CookiePLMonster** [comment]
First of all, that's why the patch is in beta - I fixed what I could, but there may be more lingering issues.

Second, to verify if you've installed it correctly, check if the version string shown in the main menu is `1.200 SP Build 3`.

**Comment by howiplay1** [comment]
fuck you man dont release something as a "patch" that doesnt work, that literally makes it bloatware

**Comment by Skibbles-FSE** [comment]
@howiplay1 well that isn't very polite.

**Comment by xamzy** [comment]
@howiplay1 This most certainly does work, and I can show you that it does, instead of you spreading false information. 

This has been documented by many people, most popular of all being Sweg: https://youtu.be/AAMmHhirmo0 

This has been used for speedrunning, by speedrunners since its release, see here: 
https://www.speedrun.com/bullyse/thread/o9i7a 

and here:

**Comment by xamzy** [comment]
@howiplay1 This most certainly does work, and I can show you that it does, instead of you spreading false information. 

This has been documented by many people, most popular of all being Sweg: https://youtu.be/AAMmHhirmo0 

This has been used for speedrunning, by speedrunners since its release, see here: 
https://www.speedrun.com/bullyse/thread/o9i7a 

and here: https://youtu.be/EGQHBAg5YtU - where i play the game full 100% and you can see it in the main menu at times where I reboot the game. 

and here: https://github.com/CookiePLMonster/SilentPatchBully/blob/master/SilentPatchBully/SilentPatchBully.cpp - which is the code directly responsible for patching the game, with notes detailing what each thing is directly doing. 

It is not our fault that you are incompetant, and unable to utilise this. So don't say "it doesn't work" and that it's "bloatware" when it 
A: does work 
B: doesn't bloat anything. 

Grow up.

**Comment by Skibbles-FSE** [comment]
@howiplay1 However I would suggest you look into other avenues of solving your issues regarding the crashes, it is very likely you are encountering some other issue that is tied to the machine it is running on/the variables it is running under. It's important to not get upset and frustrated at those who wish only to help you and others. Try googling your issue and look from there, this place is not the best area to diagnose it.

**Comment by xamzy** [comment]
@Skibbles-FSE incorrect, this is the best place to be asking and diagnosing, however the way he is going about it is not the best.

**Comment by Skibbles-FSE** [comment]
@xamzy Oh, I figured this was looking for issues tied to the patch directly versus his system itself. Apologies.

**Comment by xamzy** [comment]
@Skibbles-FSE
 Oh, it is tied to the patch directly, however if he wants help with getting the Patch to work on his machine, he can send it crash-dumps, show what he's done so far and why he thinks it doesn't work etc, and we'll be more than happy to help :)

**Comment by Fco1998** [comment]
Hi, how are you?

Every time I play Bully, the game crashes every 1.5 hours, up to 2 hours, and hangs until it creates a .dmp file. I use the program "WinDbg" to see what has failed and it tells me "SilentPatch.asi and Access violation - code c0000005".

Is there any chance that you still receive .dmp files for further review? On the other hand, will there be any future update of the patch?

Greetings and good work. :)
